openpathsampling/collectivevariables/plumed_wrapper.py

```diff
@@ -27,7 +27,7 @@ class PLUMEDCV(paths.CoordinateFunctionCV):
     https://doi.org/10.1016/j.cpc.2013.09.018
 
     Examples
-    --------
+
```

Comment:
```suggestion
    --------
```

The original was correct. Examples is a valid numpydoc section, as shown in the [numpydoc example](https://numpydoc.readthedocs.io/en/latest/example.html).

I personally don't use it very often because most examples are either so trivial or so complicated that I don't find them informative.

Reply:
good catch, that is an accidental delete.